steps/http/model/request.go

```diff
@@ -40,7 +40,8 @@ func NewRequest(
 	request.Method = method
 	request.Endpoint = url + normalizeEndpoint(endpoint, endpoint, backslash)
 	request.RequestBody = nil
-	request.Path = path
+	request.Path = ""
+	request.QueryParams = nil
```

Comment:
Also nil, QueryParams and RequestBody, unless we are giving nil a logic 👍🏽

Reply:
Done